www/getting-started.rst

```diff
@@ -2,151 +2,80 @@
 
-* There is a sizeable collection of both generic and
-  application-specific numerical and scientific code, written using
-  Python, NumPy and SciPy. Don't reinvent the wheel, there may already
-  be a pre-made solution for your problem. See
-  :doc:`topical-software` for a partial list.
+Hello SciPy 
+===========
 
-* As Python is a popular general-purpose programming language, it has
-  many advanced modules for building for example interactive
-  applications (see e.g. wxPython_ and Traits_) or web sites (see
-  e.g. Django_). Using SciPy with these is a quick way to build a
-  fully-fledged scientific application.
+Need to test if the packages got installed?  Type these lines at an IPython prompt, or save in a `*.py` file to execute::
 
-How to work with SciPy
-----------------------
-Python is a programming language, and there are several ways to
-approach it. There is no single program that you can start and that
-gives an integrated user experience.  Instead, there are several
-possible ways to work with Python.
+    import numpy as np
+    print("I like ",np.pi)
 
-The most common is to use the advanced interactive Python shell
-IPython_ to enter commands and run scripts. Scripts can be written
-with any text editor, for instance Emacs_, Vim_ or even Notepad.  Some
-of the packages such as *Python(x,y)* mentioned in :doc:`install` also
-offer an integrated scientific development environment.
+For testing the SciPy Library and Matplotlib, here's a fun Easter Egg::
 
-Neither SciPy nor NumPy provide plotting functions. There are several
-plotting packages available for Python, the most commonly used one being
-matplotlib_.
+    import scipy.misc
+    import matplotlib.pyplot as plt
 
-Learning to work with SciPy
----------------------------
+    face=scipy.misc.face()
+    plt.imshow(face)
+    plt.show()
 
-To learn more about the Python language, the `official Python tutorial`_ is
-an excellent way to become familiar with the Python syntax and objects.
 
-One way of getting a handle on the scientific computation tools in Python
-is to take a look at the following online resources:
+Start Learning
+==============
 
-- `Python Scientific Lecture Notes <http://www.scipy-lectures.org/index.html>`_
+Each package has official tutorials:
 
 - `NumPy User Guide <http://www.numpy.org/devdocs/user/>`_
 
 - `SciPy Tutorial <http://docs.scipy.org/doc/scipy/reference/tutorial/index.html>`_
-  contains examples for each submodule in the SciPy library
 
-- `Matplotlib beginner's guide <http://matplotlib.org/users/beginner.html>`__
+- `Matplotlib beginner's guide <http://matplotlib.org/users/beginner.html>`_
 
-- `Pandas tutorials <http://pandas.pydata.org/pandas-docs/stable/tutorials.html>`__
+- `Pandas tutorials <http://pandas.pydata.org/pandas-docs/stable/tutorials.html>`_
 
-- `Sympy tutorial <http://docs.sympy.org/latest/tutorial/>`__
+- `Sympy tutorial <http://docs.sympy.org/latest/tutorial/>`_
```

Comment:
let's fix the capitalization while we're at it (`SymPy`)

Reply:
did that.